core/src/main/groovy/noe/tomcat/configure/ConnectorTomcatAbstract.groovy

```diff
@@ -103,4 +109,19 @@ abstract public class ConnectorTomcatAbstract<E extends ConnectorTomcatAbstract>
     return (E) this
   }
 
+  public E setUpgradeProtocol(UpgradeProtocolTomcat upgradeProtocol = new UpgradeProtocolTomcat()) {
```

Comment:
Could we write it in Java standard way? Because this way is a convention in Tomcat configuration code, let's please stick with that.